lib/src/assets_audio_player.dart

```diff
@@ -595,7 +596,11 @@ class AssetsAudioPlayer {
           "playSpeed": playSpeed ?? this.playSpeed.value ?? defaultPlaySpeed,
         };
         if (seek != null) {
-          params["seek"] = seek.inSeconds.round();
+          if (Platform.isAndroid) {
+            params["seek"] = seek.inMilliseconds.round();
+          } else {
+            params["seek"] = seek.inSeconds.round();
```

Comment:
I think it would be better to always send it as milliseconds, then depending on the platform, change it to seconds (ex: ios)

If possible I prefer to not have `Platform.isAndroid` in this package :)